src/main/steps/common/task-list/components/tasklist/utils.ts

```diff
@@ -139,6 +141,14 @@ const taskListConfig = {
             show: (caseData: Partial<CaseWithId>): boolean => isActiveCase(caseData),
             stateTag: () => StateTags.SUBMITTED,
           },
+          {
+            id: Tasks.SUPPORT_DURING_CASE,
+            href: () => {
+              return `${APPLICANT_TASKLIST_HEARING_NEEDS}`;
+            },
+            show: (caseData: Partial<CaseWithId>): boolean => isActiveCase(caseData),
```

Comment:
isActiveCase is no longer valid.

Reply:
Should it be removed? It's mentioned in all other tasks, that's why I included it.